yelp_beans/logic/meeting_spec.py

```diff
@@ -55,7 +55,14 @@ def get_users_from_spec(meeting_spec):
     return users
 
 
-def get_meeting_datetime(meeting_spec):
+def get_meeting_datetime(meeting_spec, user=None):
+    ''' Get the meeting datetime for user. If user is specified, the timezone will be the user's
+    timezone preference. If not specified, the timezone will be the meeting spec's timezone.
+    '''
+    if user and user.timezone:
```

Comment:
Agreed, we should version.  

Though I am not sure that addresses the problem here.  I think that if we have old records that don't have the timezone entry (our current prod instance) and we update the model and attempt to access user.timezone, it will say that timezone does not exist on the user even though the model is defined properly.